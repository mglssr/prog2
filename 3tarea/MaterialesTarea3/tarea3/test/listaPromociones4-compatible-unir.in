crearListaPromociones
crearPromocion 01/01/2002 01/07/2002 10 1
agregarAPromocion 9 0 1 2 3 4 5 6 7 8
agregarAListaPromociones
crearPromocion 01/02/2002 01/07/2002 10 2
agregarAPromocion 9 0 1 2 3 4 5 6 7 8
agregarAListaPromociones
crearPromocion 02/03/2002 01/07/2002 10 3
agregarAPromocion 9 0 1 2 3 4 5 6 7 8
agregarAListaPromociones
imprimirListaPromociones
crearPromocion 03/03/2003 01/07/2003 10 3
agregarAPromocion 9 0 1 2 3 4 5 6 7 8
esCompatibleListaPromociones
liberarPromocion
crearPromocion 02/03/2002 01/07/2002 10 3
agregarAPromocion 9 0 1 2 3 4 5 6 7 8
esCompatibleListaPromociones
liberarPromocion
crearPromocion 02/03/2002 01/07/2002 10 3
agregarAPromocion 1 9
esCompatibleListaPromociones
liberarPromocion
unirListaPromociones 2 04/03/2002 01/07/2002 10 4 2 3 4 05/03/2002 01/07/2002 10 5 2 2 3
liberarListaPromociones
Fin
