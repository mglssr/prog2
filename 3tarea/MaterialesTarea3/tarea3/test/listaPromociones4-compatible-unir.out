Bienvenido al programa principal de MercadoFinger. Por favor ingrese su comando:
1> Lista de promociones creada con exito.
2> Promocion creada con exito.
3> Productos agregados a la promocion de forma exitosa.
4> Se ha agregado la promocion a la lista de forma exitosa.
5> Promocion creada con exito.
6> Productos agregados a la promocion de forma exitosa.
7> Se ha agregado la promocion a la lista de forma exitosa.
8> Promocion creada con exito.
9> Productos agregados a la promocion de forma exitosa.
10> Se ha agregado la promocion a la lista de forma exitosa.
11> Promocion #1 del 1/1/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 
Promocion #2 del 1/2/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 
Promocion #3 del 2/3/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 
12> Promocion creada con exito.
13> Productos agregados a la promocion de forma exitosa.
14> La promocion es compatible con el resto de las promociones de la lista.
15> Promocion liberada con exito.
16> Promocion creada con exito.
17> Productos agregados a la promocion de forma exitosa.
18> La promocion NO es compatible con el resto de las promociones de la lista.
19> Promocion liberada con exito.
20> Promocion creada con exito.
21> Productos agregados a la promocion de forma exitosa.
22> La promocion es compatible con el resto de las promociones de la lista.
23> Promocion liberada con exito.
24> Promocion #1 del 1/1/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 
Promocion #2 del 1/2/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 
Promocion #3 del 2/3/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 
Promocion #4 del 4/3/2002 al 1/7/2002
Productos: 3 4 
Promocion #5 del 5/3/2002 al 1/7/2002
Productos: 2 3 
25> Lista promociones liberada con exito.
26> Fin.
